pkg/apis/deployment/deployment_test.go

```diff
@@ -644,16 +643,7 @@ func TestObjectSynced_MultiJob_ContainsAllElements(t *testing.T) {
 				assert.Equal(t, "edcradix-machine-user", getRoleBindingByName("radix-app-adm-job", rolebindings).Subjects[1].Name)
 
 				// Exists due to being job-scheduler
-				assert.True(t, roleBindingByNameExists(defaults.RadixJobSchedulerEnvRoleName, rolebindings), "Expected rolebinding radix-job-scheduler-env to be there to access secrets, RadixBatches, etc")
-			})
-
-			t.Run(fmt.Sprintf("%s validate rolebindings in app namespace", testScenario), func(t *testing.T) {
-				t.Parallel()
-				rolebindings, _ := kubeclient.RbacV1().RoleBindings(appNamespace).List(context.TODO(), metav1.ListOptions{})
-				assert.Equal(t, 1, len(rolebindings.Items), "Number of rolebindings was not expected")
-
-				// Exists due to being job-scheduler
-				assert.True(t, roleBindingByNameExists(defaults.RadixJobSchedulerAppRoleName, rolebindings), "Expected rolebinding radix-job-scheduler-app to be there to access RadixApplication")
+				assert.True(t, roleBindingByNameExists(defaults.RadixJobSchedulerRoleName, rolebindings), "Expected rolebinding radix-job-scheduler-env to be there to access secrets, RadixBatches, etc")
```

Comment:
```suggestion
				assert.True(t, roleBindingByNameExists(defaults.RadixJobSchedulerRoleName, rolebindings), "Expected rolebinding radix-job-scheduler-env to be there to access secrets, RadixBatches, etc")
```
assert.True(t, roleBindingByNameExists(defaults.RadixJobSchedulerRoleName, rolebindings), "Expected rolebinding radix-job-scheduler to be there to access secrets, RadixBatches, etc")